pkg/controllers/vdb/imageversion_reconciler.go

```diff
@@ -108,6 +109,17 @@ func (v *ImageVersionReconciler) Reconcile(ctx context.Context, _ *ctrl.Request)
 	return v.verifyNMADeployment(vinf, pod)
 }
 
+// genSandboxOldVersionInfo will build and return the sandbox in the configmap
+// annotations
+func (v *ImageVersionReconciler) genSandboxOldVersionInfo(ctx context.Context) (*version.Info, error) {
+	sbTrigger := MakeSandboxTrigger(v.Rec, v.Vdb, v.PFacts.SandboxName, "" /*no uuid*/)
+	oldVersion, err := sbTrigger.getSandboxVersion(ctx)
```

Comment:
Using the sandbox trigger to fetch the version is an odd choice. I guess it has the convenience that it already knows how to fetch the configMap. Is there a better way to do that?

Reply:
How about renaming sandbox trigger?

Reply:
Yes, that works too.